codex/merkletree/merkletree.nim

```diff
@@ -0,0 +1,111 @@
+import std/sequtils
+import std/math
+import std/bitops
+import std/sugar
+
+import pkg/libp2p
+import pkg/stew/byteutils
+
+import pkg/questionable
+import pkg/questionable/results
+
+
+type
+  MerkleHash* = MultiHash
+  MerkleTree* = ref object of RootObj
```

Comment:
Does this need to be a type of `RootObj`? Not sure we'll need methods for this.

Reply:
Not unless we're planing to extend this, which we are not. We want this to just be an object.

```nim
  MerkleTree* = object
```

Reply:
Thanks for the info guys, changing it to `object`.